R/executeDqChecks.R

```diff
@@ -127,7 +132,10 @@ executeDqChecks <- function(connectionDetails,
     metadata$dqdVersion <- as.character(packageVersion("DataQualityDashboard"))
     DatabaseConnector::disconnect(connection)
   } else {
-    metadata <- NA
+    metadata <- data.frame(
+      DQD_VERSION = as.character(packageVersion("DataQualityDashboard")),
```

Comment:
here and throughout the PR, please convert all variable names to camelcase; this is a required convention for HADES packages

Reply:
Which variables do you want as camelCase?  `metadata` is a pre-existing variable.

Reply:
I think I figured out what you want.  Done

Reply:
Thanks!  These 2 variables in the metadata should also be camelcase :)